orttraining/orttraining/training_ops/cuda/optimizer/common.cuh

```diff
@@ -15,11 +15,12 @@ namespace cuda {
 template<typename TLossScale, typename TGradNorm, typename TFinalScale>
 __device__ __forceinline__ TFinalScale _ComputeGradScale(
 const TLossScale* loss_scale,
-const TGradNorm* g_norm) {
+const TGradNorm* g_norm,
+const TFinalScale max_norm) {
 TFinalScale scale = loss_scale != nullptr ? TFinalScale(*loss_scale) : TFinalScale(1.f);
```

Comment:
we referece https://pytorch.org/docs/stable/_modules/torch/nn/utils/clip_grad.html#clip_grad_norm_ for the clipping.